index.js

```diff
@@ -16,4 +16,3 @@ module.exports = function is(a, b) {
 	}
 	return false;
 };
-
```

Comment:
_without_ this change linting is broken. I have to do this to make tests passed.